### ISSUE TITLE
Ultrawide doesn't work at first until you switch to windowed and then back to fullscreen.

### ISSUE THREAD

**Issue by Demetter** [cathyjf/SekiroFpsUnlockDll#2]
Ultrawide doesn't work when you first start the game, you have to switch to windowed mode using the ingame settings and then switch back to fullscreen for the ultrawide to work, otherwise there's black bars making it 16:9. 

It's not the end of the world but its kind of annoying, please fix if you can.

**Comment by neolithic-computer** [comment]
I agree, it's a minor annoyance. The same thing happens with the executable program that this project is based upon ([uberhalit/SekiroFpsUnlockAndMore](https://github.com/uberhalit/SekiroFpsUnlockAndMore)), so my guess is that the original author would need to address this, which unfortunately seems unlikely at this point.

**Comment by fm541923607** [comment]
the same issue to me, I hope author need to annouce in the readme if have no time for this. there is another way can fix it, as it mentioned in another issue report, change nvidia control pannel v-sync from open to fast, it also will work. hope author can add this comment in readme for ultrawide users. thank you very much for your hard working.

**Comment by cathyjf** [comment]
I don't have access to an ultrawide display, so I can't verify this issue or debug it.
